doc/source/configuration.rst

```diff
@@ -163,12 +161,18 @@ The Nextflow configuration file is used to configure various settings for the *k
         
       Has to match enabled container type (see above).
 
-    * **withName** defines settings for individual processes in the *ksrates* Nextflow pipeline, for example:
+    * **withName** defines settings for individual processes in the *ksrates* Nextflow pipeline.
+    
+      There are 11 processes in the pipeline, 6 of which (``checkConfig``, ``setupCorrection``, ``setParalogAnalysis``, ``setOrthologAnalysis``,  ``doRateCorrection`` and ``drawTree``) are by default run locally because they execute minimal calculations. The remaining 5 processes (``estimatePeak``, ``plotOrthologDistrib``, ``paralogsAnalyses``, ``wgdParalogs`` and ``wgdOrthologs``) are instead run by default on a cluster, if available, and can be configured under this section of the Nextflow configuration file. ``wgdParalogs`` and ``wgdOrthologs`` are the most computationally demanding and it is advised to assign them a higher computational power than the others (suggested: parallelization with about 10 threads and 2GB memory each).
```

Comment:
```suggestion
      There are 11 processes in the pipeline, 6 of which (``checkConfig``, ``setupCorrection``, ``setParalogAnalysis``, ``setOrthologAnalysis``,  ``doRateCorrection`` and ``drawTree``) are run locally, i.e. on the computer or compute node where the pipeline is launched, and sequentially by default because they involve minimal calculations. The remaining 5 processes (``wgdParalogs``, ``wgdOrthologs``, ``plotOrthologDistrib``, ``estimatePeak`` and ``paralogsAnalyses``) are instead run (or should be run) by default in parallel on a compute cluster, if available, which is configured under this section of the Nextflow configuration file. ``wgdParalogs`` and ``wgdOrthologs`` are the most computationally demanding processes and it is advised to assign each of these (parallel) processes a higher computational power than the other processes (suggested: about 10 CPUs/cores/threads and 2GB memory each per process).
```
With "2GB memory each" you mean per CPU/thread?

Reply:
We should probably rephrase this last sentence... I meant indeed "per thread", but we can directly say something like `suggested: about 10 CPUs/cores/threads and 20GB of memory`.

Reply:
Might be good to still define this per CPU so users know when they use a different amount. How about this:
`... than the other processes. If available, we suggest to configure about 10 CPUs/cores/slots/threads and about 20GB memory (or, on average, about 2GB per configured CPU) each for these processes).`

Reply:
What about: `If available, we suggest to configure about 10 CPUs/cores/slots/threads and about 20GB memory (or, on average, about 2GB per configured CPU) for each of these two processes.` to make more clear that only the two wgdProcesses need this amount of resources, and not the other ``plotOrthologDistrib``, ``estimatePeak`` and ``paralogsAnalyses``.

Reply:
Sure!